packages/mediacenter/LibreELEC-settings/scripts/factory-reset

```diff
@@ -38,42 +32,37 @@ get_target() {
 }
 
 show_reset_msg() {
-  echo "RESET IN PROGRESS"
-  echo ""
-  echo "Please do not reboot or turn off your ${NAME} device!"
-  echo ""
+  echo -e "RESET IN PROGRESS\n"
+  echo -e "Please do not reboot or turn off your @DISTRONAME@ device!\n"
 }
 
-if [ -f /storage/.cache/reset_oe ] ; then
+if [ -f /storage/.cache/reset_oe ]; then
   # hard reset
   rm -f /storage/.cache/reset_oe
   get_target
-  if [ ! -z $target ] ; then
+  if [ -n $target ]; then
     show_reset_msg
 
     StartProgress spinner "Performing hard reset... "
-      umount /storage
-      mke2fs -t ext4 -m 0 $target &>/dev/null
-      if [ ! -z $label ] ; then
-        tune2fs -U random -L $label $target &>/dev/null
-      fi
-      if [ ! -z $uuid ] ; then
-        tune2fs -U $uuid $target &>/dev/null
-      fi
+      rm -rf /storage/* /storage/.* &>/dev/null
+      [ $(blkid -o value -s TYPE $target) = "ext4" ] && mkdir -m 700 -p "/storage/lost+found/"
       StopProgress "done!"
 
     echo
     StartProgress countdown "Rebooting in 5s... " 5 "NOW"
   fi
-elif [ -f /storage/.cache/reset_xbmc ] ; then
+elif [ -f /storage/.cache/reset_xbmc ]; then
```

Comment:
As much as I like a trip down memory lane, perhaps we should update/deobfuscate the trigger-file names and use reset_soft/reset_hard instead of reset_oe/reset_xbmc ?

Reply:
Commit added to do this. I'll PR the necessary change to settings addon shortly.